cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1454,16 +1461,6 @@ spec:
                     iteration:
                       type: integer
                       format: int32
-            security:
```

Comment:
Does this block just need to move up to line 285?

Reply:
Right. Done